features/luks/data/lib/uevent/handlers/085-luks

```diff
@@ -48,8 +48,26 @@ match_dev_in_array() {
 	return 1
 }
 
+luks_unl0kr_ask_pass() {
+	[ "$#" = 2 ] || return 1
+
+	local cmd="$1" tries=$2 rc=2
+
+	plymouth hide-splash 2>/dev/null
+
+	while [ $rc != 0 ] && [ $tries -ne 0 ]; do
+		unl0kr 2>/dev/null | $cmd
+		rc=$?
+		tries=$(( $tries - 1 ))
```

Comment:
Please do not add extra whitespaces here.

Reply:
I got style from line [218](https://github.com/osboot/make-initrd/pull/39/files/d0adb92711b6c7628693203d6046beed12cfb259#diff-d50c296c234eaa6a3a14f88ba4dcda4fb99511e4197cfe5e174315a8f098921eR218) of the current commit.